client/implementation-vertx/src/main/java/io/smallrye/graphql/client/vertx/typesafe/VertxTypesafeGraphQLClientBuilder.java

```diff
@@ -185,7 +195,8 @@ public <T> T build(Class<T> apiClass) {
                 endpoint,
                 websocketUrl, executeSingleOperationsOverWebsocket, httpClient, webClient, subprotocols,
                 websocketInitializationTimeout,
-                allowUnexpectedResponseFields);
+                allowUnexpectedResponseFields,
+                namespaceAnnotation != null);
```

Comment:
Why would you pass a `useNamespace` boolean when you have already looked at the annotation and can now already just pass the information about the particular namespace as a `List<String>`? That would avoid having to use reflection later on.